src/bindings/PyDP/algorithms/partition_selection_strategies.cpp

```diff
@@ -0,0 +1,72 @@
+// Provides bindings for Bounded Functions
+
+#include "pybind11/complex.h"
+#include "pybind11/functional.h"
+#include "pybind11/pybind11.h"
+#include "pybind11/stl.h"
+
+#include "algorithms/partition-selection.h"
+
+using namespace std;
+
+namespace py = pybind11;
+namespace dp = differential_privacy;
+
+typedef std::unique_ptr<dp::PartitionSelectionStrategy> PyPartitionSelectionStrategy;
+
+PyPartitionSelectionStrategy create_truncted_geometric_partition_strategy(
```

Comment:
Nit: CreateTruncatedGeometric....
https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

(as far as I understand C++ names and Python names can be different)